api/controllers/v1/massif/update.js

```diff
@@ -0,0 +1,80 @@
+const ControllerService = require('../../../services/ControllerService');
+const ErrorService = require('../../../services/ErrorService');
+const MappingService = require('../../../services/MappingService');
+const RightService = require('../../../services/RightService');
+const MassifService = require('../../../services/MassifService');
+// eslint-disable-next-line consistent-return
+module.exports = async (req, res) => {
+  // Check right
+
+  const hasRight = await sails.helpers.checkRight
+    .with({
+      groups: req.token.groups,
+      rightEntity: RightService.RightEntities.MASSIF,
+      rightAction: RightService.RightActions.EDIT_ANY,
+    })
+    .intercept('rightNotFound', () =>
+      res.serverError(
+        'A server error occured when checking your right to update a caver.'
+      )
+    );
+  if (!hasRight) {
+    return res.forbidden('You are not authorized to update a massif.');
+  }
+
+  // Check if massif exists
+  const massifId = req.param('id');
+  if (
+    !(await sails.helpers.checkIfExists.with({
+      attributeName: 'id',
+      attributeValue: massifId,
+      sailsModel: TMassif,
+    }))
+  ) {
+    res.status(404);
+    return res.json({ error: `Could not find massif with id ${massifId}.` });
+  }
+
+  // list of propreties to update
+  const propretiesUpdatable = [
+    'caves',
+    'descriptions',
+    'documents',
+    'geogPolygon',
+    'names',
+  ];
+  // Check if the changes requested are authorized (check propretiesUpdatable)
+  const keys = Object.keys(req.body);
+  // eslint-disable-next-line consistent-return
```

Comment:
Pas de `eslint disable`, une simple boucle for ou while suffit ici pour l'éviter.